src/foam/u2/TableView.js

```diff
@@ -46,6 +49,14 @@ foam.CLASS({
   methods: [
     function addObj(obj) {
       var e = this.start('tr')
+          .on('click', function() {
+            this.tableView.selection = obj;
```

Comment:
Each extra listener is 11k. So if you're scrolling 100 rows at 60fps that's ~ 10k*100rows*60frames = 60000 or 60M / sec